src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -167,7 +167,9 @@
     <comment>0 - file path ending in _._</comment>
   </data>
   <data name="InvalidPrereleaseDependencyWarning" xml:space="preserve">
-    <value>A stable release of a package should not have a prerelease dependency. Either modify the version spec of dependency "{0}" or update the version field in the nuspec.</value>
+    <value>A stable release of a package should not have a prerelease dependency for "{0}". Either modify the version spec of dependency "{1}" or update the version field in the nuspec.</value>
```

Comment:
Is this message going to make sense for customers who have a custom `nuspec` and use a `dependencies` node without a `framework=` attribute?

Reply:
Reverted.